lib/src/components/NxLoadError/__tests__/NxLoadError.test.tsx

```diff
@@ -4,86 +4,99 @@
 
   it('does not render anything if error is unset', function() {
-    expect(getShallowComponent()).toBeEmptyRender();
+    expect(renderEl()).not.toBeInTheDocument();
   });
 
   it('renders an error alert when error is set', function() {
-    expect(getShallowComponent({ error: 'Error!' }))
-        .toContainMatchingElement(NxErrorAlert);
+    const el = renderEl({ error: 'Error!' });
+    expect(el).toHaveAttribute('role', 'alert');
   });
 
   it('passes a default title message if there is an error', function() {
-    expect(getShallowComponent({ error: 'Error!' }).render().text())
-        .toContain('An error occurred loading data');
+    const el = renderEl({ error: '' });
+    expect(el?.textContent).toContain('An error occurred loading data. ');
   });
 
   it('uses the specified title message instead of the default', function() {
-    expect(getShallowComponent({ error: 'Error!', titleMessage: 'This is bad' }).render().text())
-        .not.toContain('An error occurred loading data');
+    const el = renderEl({ error: '', titleMessage: 'This is bad!' });
+    expect(el?.textContent).not.toBe('An error occurred loading data. ');
+    expect(el?.textContent).toContain('This is bad! ');
 
-    expect(getShallowComponent({ error: 'Error!', titleMessage: 'This is bad' }).render().text())
-        .toContain('This is bad');
   });
 
   it('Uses the error as children', function() {
-    expect(getShallowComponent({ error: 'Server Error', titleMessage: 'This is bad!' }).render().text())
-        .toContain('Server Error');
+    const el = renderEl({ error: 'Server Error', titleMessage: 'This is bad Error!' });
+    expect(el?.textContent).toContain('Server Error');
   });
 
   it('renders the full error message', function() {
-    const shallowComponent = getShallowComponent({ error: 'Server Error', titleMessage: 'This is bad!' }),
-        childrenText = shallowComponent.render().text();
-
-    expect(childrenText).toEqual('This is bad! Server Error');
+    const el = renderEl({ error: 'Server Error', titleMessage: 'This is bad!' });
+    expect(el?.textContent).toEqual('This is bad! Server Error');
   });
 
   it('renders a retry button if there is an error and retryHandler is set', function() {
-    expect(getShallowComponent({ error: 'Error!' })).not.toContainMatchingElement(NxButton);
-
-    expect(getShallowComponent({ error: 'Error!', retryHandler: () => {} })).toContainMatchingElement(NxButton);
-    expect(getShallowComponent({ error: 'Error!', retryHandler: () => {} }).find(NxButton)).toIncludeText('Retry');
+    const elWithoutRetryButton = renderEl({ error: 'Error' });
+    expect(elWithoutRetryButton?.textContent).not.toContain('Retry');
+    const elRetryButton = renderEl({ error: 'Error', retryHandler: () => {} });
+    const retryButton = screen.getByRole('button', {name: 'Retry'});
+    expect(elRetryButton?.textContent).toContain('An error occurred loading data. Error');
```

Comment:
This line doesn't need to be part of this test

Reply:
> This line doesn't need to be part of this test

Can you, please, specify what kind of line you mean?  Is it something wrong in this code?

Reply:
> This line doesn't need to be part of this test

And I change it previous comment, is it fixed now? ```
it('renders a retry button if there is an error and retryHandler is set', async function() {
    const elWithoutRetryButton = renderEl({ error: 'Error' });
    expect(elWithoutRetryButton?.textContent).not.toContain('Retry');
    expect(quickRender({ error: 'Error', retryHandler: () => {} }).getByRole('button', { name: 'Retry' }))
        .toBeInTheDocument();
  });
```

Reply:
> This line doesn't need to be part of this test

And I change it previous comment, is it fixed now? 
```
it('renders a retry button if there is an error and retryHandler is set', async function() {
const elWithoutRetryButton = renderEl({ error: 'Error' });
expect(elWithoutRetryButton?.textContent).not.toContain('Retry');
expect(quickRender({ error: 'Error', retryHandler: () => {} }).getByRole('button', { name: 'Retry' }))
.toBeInTheDocument();
});
```

Reply:
> And I change it previous comment, is it fixed now?

That seems right.  My point was that checking the text content of `elRetryButton` seemed unnecessary